x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/operator/DefaultOperatorOnlyRegistry.java

```diff
@@ -37,7 +36,7 @@ public class DefaultOperatorOnlyRegistry implements OperatorOnlyRegistry {
         TransportAddVotingConfigExclusionsAction.TYPE.name(),
         TransportClearVotingConfigExclusionsAction.TYPE.name(),
         PutLicenseAction.NAME,
-        DeleteLicenseAction.NAME,
```

Comment:
Meh, this broke rest tests that used the constant from the core compile-only dependency since it is inlined at compile time afaik, but this stopped working when moving to `TYPE.name()`. => I inlined the string manually here instead of messing with the dependency structure for now.